src/Speckle.Objects/Geometry/Polyline.cs

```diff
@@ -12,170 +10,29 @@ namespace Speckle.Objects.Geometry;
 /// A polyline curve, defined by a set of vertices.
 /// </summary>
 [SpeckleType("Objects.Geometry.Polyline")]
-public class Polyline : Base, ICurve, IHasArea, IHasBoundingBox, IConvertible, ITransformable
```

Comment:
I approve of removing I convertible, but I'm fairly certain this was being done for to allow a Polyline to be casted to a PolyCurve in some cases?

Obviously this was not the right way to do this either, so the removal is good, but if this is needed maybe we should implement explicit casting instead.

Reply:
I don't think we were ever using the IConvertible ness...

You would have had to either manually call ToType<Polyline>(), or use the `[ChangeType](https://learn.microsoft.com/en-us/dotnet/api/system.convert.changetype?view=net-8.0#system-convert-changetype(system-object-system-type)) helper function...

I don't think casts are appropriate either, but I would accept a `ToPolyLine` function or similar... if we need it that is.